docs/design/coreclr/botr/readytorun-format.md

```diff
@@ -428,6 +485,31 @@ The module override index translation algorithm is as follows (**ILAR** = *the n
 
 **TODO**: document attribute presence encoding
 
+**Note**: We already know this table uses assembly-relative token encoding so it has similar
```

Comment:
I am not sure what this is trying to say. Should this just say: "**Note:** This is a per-assembly section. ... " ?

Reply:
I guess I was trying to say "I'm not sure" at the time of writing the document. As you confirm my suspicion that this is a per-assembly table, I have changed it as you suggested in the 3rd commit.